src/block/icon-list/edit.js

```diff
@@ -60,16 +60,12 @@ const includeEditorContentAlignClassName = attributes => {
 
 	const addedAlignmentClass = {}
 
-	if ( getPreviewDeviceType ) {
-		const previewDeviceType = getPreviewDeviceType()
-		const alignment = (
-			previewDeviceType === 'Desktop' ? contentAlign :
-				previewDeviceType === 'Tablet' ? tabletContentAlign :
-					mobileContentAlign
-		) || 'left'
+	const previewDeviceType = getPreviewDeviceType ? getPreviewDeviceType() : 'Desktop'
+	const alignment = (
+		previewDeviceType === 'Desktop' ? contentAlign : previewDeviceType === 'Tablet' ? tabletContentAlign : mobileContentAlign
+	) || 'left'
 
-		addedAlignmentClass[ `ugb-icon-list__${ alignment }-align` ] = alignment
-	}
+	addedAlignmentClass[ `ugb-icon-list__${ alignment }-align` ] = alignment
```

Comment:
Please move all the WP 5.4 `getPreviewDeviceType` compatibility to another PR so we can have the global settings PR only about global settings